gateway/api/serializers.py

```diff
@@ -168,6 +168,7 @@ class JobSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Job
+        fields = ["id", "result", "status", "program", "created"]
```

Comment:
This is not needed because fields should be defined in its V1 Serializer:

https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/v1/serializers.py#L156

To use that serializer you need to create an static method, Aki like here:
https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/v1/views.py#L38

And you can override it here:
https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/views.py#L93

This is the way that we are managing the different serializers.